setup.py

```diff
@@ -90,7 +90,8 @@ def read(fname):
         "Intended Audience :: System Administrators",
```

Comment:
Same thing here, change doesn't match easybuild 507.

Reply:
@akesandgren Those additional classifiers were only added in the `4.x` branch (for the upcoming EasyBuild 4.0).

Here, I'm just fixing the Python version (like we also did in `develop` in the `easybuild` repo, see https://github.com/easybuilders/easybuild/pull/506/files)

Reply:
okidoki, too many PRs too keep track of :-)